### DISCUSSION TITLE
水

### DISCUSSION THREAD

**Discussion by xiaoxi369258** [share121/inter-knot#4521] [灌水]
# 提问
***新艾利都一共有多少个邦布需要被救援?***
![邦布](https://upload-bbs.miyoushe.com/upload/2022/08/08/285100810/fa6e417c5c75e3b22f2ad31d899fc27d_5557053113973967857.gif)

**Comment by xiaoxi369258** [comment]
text

**Comment by obyz-wzr** [comment]
114514个
